packages/components/time-picker/__tests__/time-picker.test.ts

```diff
@@ -835,4 +836,62 @@ describe('TimePicker(range)', () => {
     expect(startInput.element.value).toBe('')
     expect(endInput.element.value).toBe('')
   })
+
+  it('can auto skip when arrow control', async () => {
```

Comment:
This test probably should not be placed inside `describe('TimePicker(range)')`.

Maybe move it to the `describe('TimePicker')`